fury/ui/tests/test_core.py

```diff
@@ -6,6 +6,7 @@
 from fury.data import DATA_DIR, read_viz_icons, fetch_viz_icons
 from fury import window, ui
 from fury.testing import EventCounter
+import numpy as np
```

Comment:
can you update import order

Reply:
Done with [c43adff](https://github.com/fury-gl/fury/pull/441/commits/c43adff5b8f7f21d539bf58dc8e2f12d4d0cf640).